BraveRewardsUI/Settings/Auto-Contribute/Details/AutoContributeDetailsViewController.swift

```diff
@@ -382,3 +386,47 @@ extension AutoContributeDetailViewController {
     }
   }
 }
+
+/// Ledger Observers
+extension AutoContributeDetailViewController {
+  func setObservers() {
```

Comment:
Assuming this is only ever done once, lets call it `setupLedgerObservers`

Also lets have this setup in `init`, and add `guard viewIsLoaded else { return }` before any part of the methods require the view constructed.  In this case below its going to be at the top